src/components/Notifications/NotificationItemImage.types.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+type NotificationItemImageProps = {
+  backgroundColor?: string
+  image: string | React.ReactNode
+  icon?: React.ReactNode
+}
```

Comment:
What if we change these props to:
```suggestion
type NotificationItemImageProps = {
  badgeIcon?: React.ReactNode
  imageBackgroundColor?: string
  image: string | React.ReactNode
}
```